cloud-sql/mysql/client-side-encryption/tests/test_encrypt_and_insert_data.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
```

Comment:
In our Python samples, we keep the tests alongside the files they're testing. Please move this to the `snippets/` directory and rename it to `encrypt_and_insert_data_test.py`